src/CloudNimble.DotNetDocs.Sdk/Sdk/Sdk.targets

```diff
@@ -129,12 +116,13 @@
             ShowPlaceholders="$(ShowPlaceholders)">
             <Output TaskParameter="GeneratedFiles" ItemName="GeneratedDocumentationFiles" />
         </CloudNimble.DotNetDocs.Sdk.Tasks.GenerateDocumentationTask>
+        
+        <ItemGroup>
+            <FileWrites Include="@(GeneratedDocumentationFiles)" />
```

Comment:
adding 'FileWrites' makes `dotnet clean` work with your doc generation. If you could 'precompute' the list of expected outputs from doc generation, then you could make doc generation zero-cost if none of the inputs and none of the outputs have changed. The power of MSBuild!